docs/source/gbd2019_models/hemoglobin_anemia_and_iron_deficiency/hemoglobin/index.rst

```diff
@@ -6,4 +6,290 @@ Hemoglobin Model
+
+Vivarium Modeling Strategy
+--------------------------
+
+Scope
++++++
+
+The Vivarium hemoglobin modeling strategy will be to sample values from the GBD 2019 year-, location-, age-, and sex-specific hemoglobin concentration distribution in order to assign specific hemoglobin concentration values to individual simulants. 
+
+Restrictions
+++++++++++++
+
+There are no formal restrictions; however, the hemoglobin estimates are unreliable for the early and late neonatal age groups and should be used with caution. Additionally, the pregnancy adjustment will apply to women of reproductive age, so special attention should be paid when modelling this demographic group.
+
+.. list-table:: GBD 2019 Risk Exposure Restrictions
+   :widths: 15 15 20
+   :header-rows: 1
+
+   * - Restriction Type
+     - Value
+     - Notes
+   * - Male only
+     - False
+     -
+   * - Female only
+     - False
+     - Note: pregnancy adjustment will apply to women of reproductive age
+   * - Age group start
+     - 0 (age_group_id=2)
+     - Note: hemoglobin estimates unreliable for early and late neonatal age groups (age_group_id=[2,3])
+   * - Age group end
+     - 95+
+     - Note: Pregnancy adjustment will apply to women of reproductive age (ages 10-55 years, age_group_ids=7-15)
+
+Assumptions and Limitations
++++++++++++++++++++++++++++
+
+.. todo::
+
+  List assumptions and limitations
+
+Data Description Tables
++++++++++++++++++++++++
+
+.. list-table:: Distribution Parameters
+  :widths: 15, 30, 10
+  :header-rows: 1
+
+  * - Parameter
+    - Value
+    - Note
+  * - Hemoglobin mean
+    - MEID 10487
+    - source='epi', decom_step='step4'
+  * - Hemoglobin standard deviation
+    - MEID 10488
+    - source='epi', decomp_step='step4'
+  * - XMAX
+    - 220
+    - 
+  * - Euler's constant
+    - 0.57721566490153286060651209008240243104215933593992
+    - 
+  * - Gamma distribution weight
+    - 0.4
+    - 
+  * - Mirror Gumbel distribution weight
+    - 0.6
+    - 
+
+Below is R code written to randomly sample hemoglobin concentration values from the hemoglobin distribution parameters and constants defined in the tables above. Additionally, the code block contains functions that will evaluate the proportion of the distribution below a given threshold. This code was adapted from the GBD stash code found `here <https://stash.ihme.washington.edu/projects/MNCH/repos/anemia/browse/model/envelope>`_, specifically the *DistList_mv2p.R* and *fit_ensemblemv2p_parallel.R* files. 
+
+.. note::
+
+  Attempts to move this code to python have not been successful in validating to GBD while using the R code has been successful. The recommended approach is to use the python package `rpy2` to run this R code within python.
```

Comment:
As Kjell noted in Slack, this is no longer accurate since he and James managed to figure out the Python version here:

https://github.com/ihmeuw/vivarium_gates_lsff/blob/master/tests/lsff_iron_exposure.ipynb

I think we should leave the R code, but update the comment and add a link to this notebook. And we could either add the Python code here as well or add a Todo to add it later.